prettier.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  printWidth: 80,
+  printWidth: 120,
```

Comment:
This was a mistake 🙃 No wonder the diffs have been so big lmao I'll fix tomorrow